titus-server-federation/src/main/java/com/netflix/titus/federation/service/CellWebClientConnector.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.federation.service;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+import com.netflix.titus.api.federation.model.Cell;
+import org.springframework.web.reactive.function.client.WebClient;
+
+public interface CellWebClientConnector {
+
+    interface WebClientFactory extends Function<Cell, WebClient> {
```

Comment:
I think we need a default implementation of this for the OSS packaging, no? (not the Embedded one used in tests, but one for `Main` classes of Federation). Without one, I'd expect the `docker-compose` test to fail

Reply:
ah, maybe we are not installing a federation instance in the `docker-compose` test

Reply:
I have added a simple implementation.